### DISCUSSION TITLE
Timestamped comments and replaying in realtime

### DISCUSSION THREAD

**Discussion by ryan-lp** [Podcastindex-org/podcast-namespace#539] [Ideas]
I'm purely an offline podcast listener, and as such I would not participate in live elements, or experience live comments or boostagrams, etc. However it occurred to me last night that if all comments had timestamps, apps should play back the comments and boostagrams in realtime replay, even in offline mode. As for submitting my own comment or boostagram, as long as the comment time is stamped onto it, other apps should be able to show the comments in realtime replay.

Hosts obviously won't receive boostagrams or see comments in realtime for a live show, but I think most listeners would still be effectively offline listeners who will listen to the podcast in their own time, so such comments would still have a benefit.

So as an example, imagine you're listening to a podcast and then something funny or maybe something controversial is said etc... I think it would be cool if in the app you could see everybody's reactions to what was said in realtime (although you're really listening to a replay of realtime). This would of course require a comments system to be more integrated than a mere linking to some external webpage.

Supposing that a podcast host has a practice of reading out every boostagram, I could imagine the software they use to receive these boostagrams could be updated to stack up all the unread boostagrams received from offline listeners over the next week, and then on the next show the host could read out the boostagrams received during the week on that next show.

**Comment by theDanielJLewis** [comment]
I would love to have this, and I included it in my proposal for cross-app comments (#505).
